test/NuGet.Core.Tests/NuGet.Versioning.Test/VersionRangeTests.cs

```diff
@@ -1503,5 +1509,29 @@ public void FindBestMatch_FloatingPrereleaseMinor_WithExtraDashes()
 
             Assert.Equal("1.9.0--beta", range.FindBestMatch(versions).ToNormalizedString());
         }
+
+        [Theory]
+        [InlineData("[1.1.4, 1.1.2)")]
+        [InlineData("[1.1.4, 1.1.2]")]
+        [InlineData("(1.1.4, 1.1.2)")]
+        [InlineData("(1.1.4, 1.1.2]")]
+        [InlineData("[1.0.0, 1.0.0)")]
+        [InlineData("(1.0.0, 1.0.0]")]
+        [InlineData("(1.0, 1.0.0]")]
+        [InlineData("[*, *)")]
```

Comment:
I sincerely hope that there are tests for this already :) `*` should not be allowed in the upper version.

Reply:
Removed.